step-templates/sitecore-ship-deploy.json

```diff
@@ -0,0 +1,82 @@
+    "Octopus.Action.CustomScripts.Deploy.ps1": "",
+    "Octopus.Action.CustomScripts.PostDeploy.ps1": ""
+  },
+  "Parameters": [
+    {
+      "Id": "8be53044-a688-48db-bc0c-d7c29c9d5993",
+      "Name": "SiteUrl",
```

Comment:
Could you please prefix all parameter names with `scs` so that we minimize the risk of clashing with other parameters in a users project.